app/controllers/cards_controller.rb

```diff
@@ -0,0 +1,30 @@
+class CardsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_game, only: [:create]
+
+  def create
+    @card = Card.new(card_params)
+    @card.author = current_user
+    @card.game = @game
+    if @card.save
+      if current_user.cards_from(@game).count < 4
+        flash[:notice] = "you must add #{ 4 - current_user.cards_from(@game).count } cards"
```

Comment:
You can pluralize this here:

`pluralize((4 - current_user.cards_from(@game)), 'card')`